doc/sphinx-guides/source/_static/api/dataset-create-software.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "http://purl.org/dc/terms/title": "Darwin's Finches",
+  "http://purl.org/dc/terms/subject": "Medicine, Health and Life Sciences",
+  "http://purl.org/dc/terms/creator": {
+      "https://dataverse.org/schema/citation/authorName": "Finch, Fiona",
+      "https://dataverse.org/schema/citation/authorAffiliation": "Birds Inc."
+  },
+  "https://dataverse.org/schema/citation/datasetContact": {
+    "https://dataverse.org/schema/citation/datasetContactEmail": "finch@mailinator.com",
+    "https://dataverse.org/schema/citation/datasetContactName": "Finch, Fiona"
+  },
+  "https://dataverse.org/schema/citation/dsDescription": {
+    "https://dataverse.org/schema/citation/dsDescriptionValue": "Darwin's finches (also known as the Galápagos finches) are a group of about fifteen species of passerine birds."
+  },
+  "datasetType": "software"
```

Comment:
Thanks, I wasn't sure where to go with this.

https://schema.org/Dataset falls under https://schema.org/CreativeWork . Should we go with that? (There's no workflow under CreativeWork, though.) 🤔

Reply:
This URI would be for the 'hasType' relationship/verb rather than the individual values. So perhaps https://schema.org/additionalType. Reading that, it looks like it allows free text types (recommending CamelCase) so would ~work given Software, Dataset, Workflow, etc. Perhaps Croissant folks or others have an opinion.

Reply:
I'm using a proper JSON-LD URL as of 19e3e51d7a.